modules/monitoring-stack-components-and-monitored-targets.adoc

```diff
@@ -56,7 +56,7 @@ In addition to the components of the stack itself, the monitoring stack monitors
 * Metering
 * OpenShift apiserver
 * OpenShift controller manager
+* Operator Lifecycle Manager (OLM)
```

Comment:
@brancz Is it alright to add OLM to the list here of cluster components the stack monitors?

cc @maximsvistunov

Reply:
@maximsvistunov was just about to add a footnote around this to make clear the monitoring team is not responsible for all components monitoring config, but generally this is good for now. I wonder if eventually we'll just remove this list and put a blanket statement saying that "everything" is monitored, which is at least the goal.

Reply:
Thanks!

FYI @ecordell @jpeeler